sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -299,9 +301,6 @@ private void suspend(boolean deleteLogs, Exception exception) {
     }
 
     private void deleteLogsOnSuspended(final GroupState groupState) {
-        if (mPersistence == null) {
```

Comment:
Why remove the null check?

Reply:
Yes, I know. this checked before. In *this* place no way to get `mPersistence == null` => no test to cover it => not 100% coverage